Dockerfile

```diff
@@ -21,5 +23,6 @@ RUN yarn install
 RUN yarn run hexo generate
 
 WORKDIR /appdata
+
 # start server
```

Comment:
Why do you have two docker files?

Reply:
Left over by accident, removed bgio.Dockerfile